src/medkit/medkit.tsx

```diff
@@ -1,68 +1,145 @@
+                <div className={'row'}>
+                    <div className="col-12">
+                        <h1>Medkit sim</h1>
+                    </div>
+                </div>
                 <div className="row">
-                    <div className="col-6">
+                    <div className="col-md-6">
                         <form>
-                            {DATA.map((el: MedkitItem, idx) => {
-                                return (
-                                    <div className={'form-check'} key={idx}>
-                                        <label className={'form-check-label'}>
-                                            <input
-                                                className={'form-check-input'}
-                                                type='checkbox'
-                                                id={String(idx)}
-                                                onChange={handlePickItem}
-                                                checked={itemsPicked[idx]}/>
-                                            Item name: {el.item} Item value: {el.value}
-                                        </label>
-                                    </div>
-                                );
-                            })}
+                            <table className={'table table-light table-sm'}>
+                                <thead>
+                                    <tr>
+                                        <th scope="col">Category</th>
+                                        <th scope="col">Item</th>
+                                        <th scope="col">Points</th>
+                                        <th scope="col">Select</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    {DATA.map((el: MedkitItem, idx) => {
+                                        return (
+                                            <tr key={idx}>
+                                                <th scope="row">{el.category}</th>
+                                                <td>{el.item}</td>
+                                                <td>{el.points}</td>
+                                                <td>
+                                                    <input
+                                                        className={'btn-check'}
+                                                        type='checkbox'
+                                                        id={String(idx)}
+                                                        onChange={handlePickItem}
+                                                        autoComplete={'off'}
+                                                        checked={itemsPicked[idx]}
+                                                        disabled={itemsPicked[idx]}/>
```

Comment:
Minor nit: The disabled style is hard to differentiate. Maybe change the color to gray? And/or the text to "added"